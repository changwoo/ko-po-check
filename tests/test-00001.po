# -*-text-*-
# test 1 - 줄임쇠(mnemonic,short cut) 검사
#
# * 줄임쇠는 대문자 이어야 한다.
# * 번역문에 괄호 안에 한 글자의 줄임쇠가 들어 있으면 줄임쇠 번역으로 본다.
#
msgid ""
msgstr ""
"Project-Id-Version: test 1\n"
"POT-Creation-Date: 2002-07-24 22:09+0900\n"
"PO-Revision-Date: 2002-07-24 22:09+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=EUC-KR\n"
"Content-Transfer-Encoding: 8bit\n"

# 대문자 그놈 줄임쇠
#: test.c:1
msgid "_File"
msgstr "파일(_F)"

# E: 소문자로 잘못 번역한 그놈 줄임쇠
#: test.c:1
msgid "T_ool"
msgstr "툴(_o)"

# 올바른 그놈 줄임쇠
#: test.c:1
msgid "Save _as..."
msgstr "다른 이름으로 저장(_A)"

# 대문자 KDE 줄임쇠
#: test.c:1
msgid "&Edit"
msgstr "편집(&E)"

# E: 소문자로 잘못 번역한 KDE 줄임쇠
#: test.c:1
msgid "C&ut"
msgstr "잘라내기(&u)"

# 올바른 KDE 줄임쇠
#: test.c:1
msgid "Pre&vious"
msgstr "이전(&V)"

# 줄임쇠가 아님
#: test.c:1
msgid "_filename"
msgstr "(_파일이름)"

# 줄임쇠가 아님 2
#: test.c:1
msgid "(test_N)"
msgstr "(테스트_N)"

# 줄임쇠가 아님 3
#: test.c:1
msgid "pre-ANSI style (K&R)"
msgstr "ANSI 이전 스타일 (K&R)"
