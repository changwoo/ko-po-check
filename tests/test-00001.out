test-00001.po:26: "(_o)": mnemonic이 소문자입니다
test-00001.po:41: "(&u)": mnemonic이 소문자입니다
test-00001.po:51: 번역문에 mnemonic이 없거나 두 개 이상입니다
